Project.toml

```diff
@@ -1,16 +1,18 @@
 name = "MKL"
 uuid = "33e6dc65-8f57-5167-99aa-e5a354878fb2"
-version = "0.5.0"
+version = "0.6.0"
 
 [deps]
 Artifacts = "56f22d72-fd6d-98f1-02f0-08ddc0907c33"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MKL_jll = "856f044c-d86e-5d09-b602-aeab76dc8ba7"
+libblastrampoline_jll = "8e850b90-86db-534c-a0d3-1478176c7d93"
```

Comment:
You aren't `using libblastrampoline_jll` directly, so I'm not sure you need this.

Reply:
Ok. Is Julia 1.8 is sufficient?

Reply:
Yes, the version of julia is basically a proxy for your `libblastrampoline_jll` requirement.  I'm not even sure the version numbers of stdlibs are actually checked with regards to compat, since some of them don't have any version number.